packages/transitive-overlay/src/util.ts

```diff
@@ -264,12 +356,13 @@ export function itineraryToTransitive(
 
   // add the routes and stops to the tdata arrays
   tdata.routes.push(...Object.values(routes));
-  tdata.stops.push(...Object.values(stops));
+  tdata.stops = newStops;
 
   // add the journey to the tdata journeys array
   tdata.journeys.push(journey);
+  tdata.places = newPlaces;
 
-  // console.log('derived tdata', tdata);
+  console.log("derived tdata", tdata);
```

Comment:
we should comment this out again!

Reply:
Yes, thanks, reverted in https://github.com/opentripplanner/otp-ui/pull/452/commits/7ca81c36b3491b584b3136e005ab964e017e0a30.